tests/test_on_data/test_imaging_interfaces.py

```diff
@@ -48,6 +48,24 @@ def check_extracted_metadata(self, metadata: dict):
         )
 
 
+@skipIf(platform.machine() == "arm64", "Interface not supported on arm64 architecture")
+class TestScanImageImagingInterfaceRecent(ImagingExtractorInterfaceTestMixin, TestCase):
+    # Second class for ScanImageImagingInterface as recommended easier than modifying the first to check metadata in
+    # each of several cases
+    data_interface_cls = ScanImageImagingInterface
+    interface_kwargs = dict(
+        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220801_volume.tif")
+    )
+    save_directory = OUTPUT_PATH
+
+    def check_extracted_metadata(self, metadata: dict):
+        assert metadata["NWBFile"]["session_start_time"] == datetime.datetime(2022, 8, 8, 16, 39, 49, 190000)
```

Comment:
```suggestion
        assert metadata["NWBFile"]["session_start_time"] == datetime(2022, 8, 8, 16, 39, 49, 190000)
```
oops

according to the CI the timestamp was a bit off